02-spatial-data.Rmd

```diff
@@ -752,7 +752,7 @@ Of course, we can represent discrete features such as soil or land-cover classes
 Consequently, the discrete borders of these features become blurred, and depending on the spatial task a vector representation might be more suitable.
 
 ```{r raster-intro-plot2, echo=FALSE, fig.cap="Examples of continuous and categorical rasters.", warning=FALSE}
-source("https://github.com/Robinlovelace/geocompr/raw/master/code/02_raster_intro_plot2.R", print.eval = TRUE)
+source("code/02_raster_intro_plot2.R", print.eval = TRUE)
```

Comment:
@Robinlovelace I would suggest going back to providing an url here. This way our readers can find the code easier. 
In terms of travis, this PR needs to be pushed to master.